### ISSUE TITLE
Make fails on macos (brew)

### ISSUE THREAD

**Issue by donomii** [cyclone-scheme/homebrew-cyclone#11]
```
2019-08-16 15:37:45 +0200

make

make: *** No rule to make target `libs/cyclone/test.c', needed by `cyclone'.  Stop.

HOMEBREW_VERSION: 2.1.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 925ec737fba1db97d0170c56de982900711cacd3
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 17c88e364dbda7a5e108341efde064b38388552a
Core tap last commit: 9 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CACHE: /Users/blah/Library/Caches/Homebrew
HOMEBREW_LOGS: /Users/blah/Library/Logs/Homebrew
CPU: octa-core 64-bit kabylake
Homebrew Ruby: 2.3.7 => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Clang: 10.0 build 1001
Git: 2.18.0 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_181
macOS: 10.14.5-x86_64
CLT: 10.3.0.0.1.1562985497
Xcode: 10.3
CLT headers: 10.3.0.0.1.1562985497
XQuartz: 2.7.11 => /opt/X11

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
CMAKE_PREFIX_PATH: /usr/local/opt/gettext:/usr/local
CMAKE_INCLUDE_PATH: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/libxml2:/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/pcre2/lib/pkgconfig:/usr/local/opt/isl/lib/pkgconfig:/usr/local/opt/mpfr/lib/pkgconfig:/usr/local/opt/ck/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.14
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk
ACLOCAL_PATH: /usr/local/opt/gettext/share/aclocal:/usr/local/share/aclocal
PATH: /usr/local/bin:/usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/gettext/bin:/usr/local/opt/pcre2/bin:/usr/local/opt/git/bin:/usr/local/opt/gcc/bin:/usr/bin:/bin:/usr/sbin:/sbin


```

**Comment by justinethier** [comment]
This should be fixed now with the latest 0.11.7 release

Note the procedure changed slightly:

* `brew tap cyclone-scheme/cyclone`
* `brew install cyclone-scheme/cyclone/cyclone-bootstrap`

**Comment by justinethier** [comment]
Independently confirmed this issue has been resolved. Closing.
